firefox-cliqz_privacy_notice/en-US.md

```diff
@@ -36,34 +36,35 @@ Firefox displays content, such as "Snippets" (messages from Mozilla) and Top Sit
 {: #snippets }
 
 ### Use Cliqz Results {: #cliqz-features }
+
 Firefox with Cliqz by default connects to Cliqz to display search results in the URL bar. Cliqz GmbH is a German company offering a privacy oriented search experience.  Learn more at their [privacy policy](https://cliqz.com/en/privacy-browser).  When the Cliqz feature is enabled:
 
 * __Search Queries and Webpage Data to Cliqz__: This includes text as you type in the URL bar, queries you send to certain search engines, and data about the webpages you visit and interactions with those pages, such as mouse movements, scrolls, and time spent.
 
 * __Interaction Data to Cliqz__: This includes your interactions with specific fields and buttons in the Cliqz feature. This data is tied to a unique identifier allowing Cliqz to understand performance over time.
 
-* __Interaction Data to Mozilla__: This includes counts of visits to search engine pages, which search engines you use, and your Cliqz identifier.  
-	
+* __Interaction Data to Mozilla__: This includes counts of visits to search engine pages, which search engines you use, and your Cliqz identifier.
+
 [Learn more](https://cliqz.com/en/whycliqz/transparency) about Cliqz data collection. To check if the Cliqz feature is enabled or to disable data collection:
-  * type "about:addons" into your Firefox URL bar
-  * select "Extensions" on the left menu bar
-  * if installed, Cliqz will appear on the right
-  * selecting "Disable" will turn off the Cliqz feature and stop related data collection
-  * selecting "Remove" will uninstall the Cliqz feature from Firefox
 
+* type "about:addons" into your Firefox URL bar
+* select "Extensions" on the left menu bar
+* if installed, Cliqz will appear on the right
+* selecting "Disable" will turn off the Cliqz feature and stop related data collection
+* selecting "Remove" will uninstall the Cliqz feature from Firefox
 
```

Comment:
This block of bullets - should the first letter in each bullet start with capital letter? It seems the style has changed compared to the others.  @brsmith-mozilla @eleemoz could you comment?

Reply:
I have not been involved with developing this notice (Elvin is on PTO this week), but I believe that each line should start with a capital letter.

Reply:
Changed.